hooks/XDR.js

```diff
@@ -0,0 +1,160 @@
+// $IIXDR,C,C,10.7,C,AIRTEMP,A,0.5,D,HEEL,A,-1.-3,D,TRIM,P,1.026,B,BARO,A,A,-4.-3,D,RUDDER*18
+
+const math = require('math-expression-evaluator');
+const fs = require('fs');
+const xdrDictionary = { definitions: [] };
```

Comment:
What's the idea with having `definitions` and not just definitions directly under root?